tasks/tests.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Test if cassandra is up
+  wait_for:
+    host: "{{ cassandra_address }}"
+    port: 9042
+
+- name: Check login with default user
```

Comment:
Hmmm, maybe we should install cqlsh only on one host (ex. first in cassandra_hosts_group`), and delegate this task to this host?
This affects all tasks using cqlsh.